python/whylogs/core/utils/stats_calculations.py

```diff
@@ -17,3 +19,31 @@ def get_distribution_metrics(
     max_val = distribution_metric.kll.value.get_max_value()
     range_val = max_val - min_val
     return min_val, max_val, range_val
+
+
+def is_probably_unique(column_profile: ColumnProfileView) -> bool:
+    cardinality = column_profile.get_metric("cardinality").to_summary_dict()
+    counts = column_profile.get_metric("counts").to_summary_dict()
+    if "cardinality" in column_profile.get_metric_names() and "counts" in column_profile.get_metric_names():
+        if cardinality["lower_1"] <= counts["n"] - counts["null"] <= cardinality["upper_1"]:
```

Comment:
Let's follow up and increase this to a few standard deviations rather than the 1 stddev in the upper_1 and lower_1 estimates.

Reply:
https://github.com/whylabs/whylogs/issues/1216